From 81e47995cc6bb6fbc62303fc9cefdba0144ff5c3 Mon Sep 17 00:00:00 2001
From: Danny Kukawka <danny.kukawka@web.de>
Date: Tue, 13 Apr 2010 15:23:17 +0200
Subject: [PATCH 30/33] add Archos 5 IMT as USB music player

Applied a fixed/slightly adopted version of a patch from
Kirk Bridger <kbridger@shaw.ca> to add Archos 5 IMT as
USB music player. (fd.o#21277)
---
 .../10freedesktop/10-usb-music-players.fdi         |   13 ++++++++++++-
 1 files changed, 12 insertions(+), 1 deletions(-)

diff --git a/fdi/information/10freedesktop/10-usb-music-players.fdi b/fdi/information/10freedesktop/10-usb-music-players.fdi
index 49bf7c3..d2f0adf 100644
--- a/fdi/information/10freedesktop/10-usb-music-players.fdi
+++ b/fdi/information/10freedesktop/10-usb-music-players.fdi
@@ -760,7 +760,18 @@
             <append key="portable_audio_player.input_formats" type="strlist">audio/x-ms-wma</append>
             <append key="portable_audio_player.input_formats" type="strlist">audio/x-wav</append>
           </match>
-        </match>
+	  <!-- Archos 5 IMT -->
+	  <match key="@storage.originating_device:usb.product_id" int="0x1332">
+            <addset key="portable_audio_player.access_method.protocols" type="strlist">storage</addset>
+            <append key="portable_audio_player.output_formats" type="strlist">audio/x-wav</append>
+            <append key="portable_audio_player.output_formats" type="strlist">audio/x-ms-wma</append>
+	    <append key="portable_audio_player.output_formats" type="strlist">application/ogg</append>
+            <append key="portable_audio_player.input_formats" type="strlist">audio/x-wav</append>
+            <append key="portable_audio_player.audio_folders" type="strlist">Music/</append>
+            <append key="portable_audio_player.playlist_format" type="strlist">audio/x-mpegurl</append>
+            <append key="portable_audio_player.playlist_path" type="strlist">Playlists/</append>
+          </match>
+	</match>
 
 	<!-- Q-be (Hyun Won, Inc.) -->
 	<match key="@storage.originating_device:usb.vendor_id" int="0x0f9c">
-- 
1.7.1

